### DISCUSSION TITLE
도대체 뭐하는 놈이냐 왜 허락없이 내  데이타베이스에 침입하여 모든정보를 엉망으로 만들어났는냐

### DISCUSSION THREAD

**Discussion by boyoun2025** [community/community#122174] [Accessibility]
### Select Topic Area

Question

### Body

도대체 뭐하는 놈이냐 왜 허락없이 내  데이타베이스에 침입하여 모든정보를 엉망으로 만들어났는냐

**Comment by boyoun2025** [comment]
boyoun2025@outlppk.com 계정 본인 입니다 본인의 컴퓨터와 핸드폰으로 지금 토론하는 곳에서 주인도 알지못하는 사이 가입하여 무슨짖을했는지 똑똑히 알려주세요 과연 진정한 주인이였다면 그 주인이 자신을 위해 이것을 사용한다면 그렇게 하였는지

**Comment by ebndev** [comment]
Hi @boyoun2025, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
